sage-ios/SAGE/SAGE/Semester.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Semester.swift
+//  SAGE
+//
+//  Created by Andrew Millman on 1/18/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+enum Term: Int {
+    case Spring = 0
+    case Fall = 1
+}
+
+class Semester: NSObject {
+    
+    var id: Int = -1
+    var startDate: NSDate?
+    var term: Term = .Spring
```

Comment:
we should have a default term value for when the data doesnt get passed back or for some other bad reason - like Term.Default